pulumi/github/teams/requirements.txt

```diff
@@ -1,2 +1,3 @@
 pulumi>=3
-pulumi_github>=5.20.0
\ No newline at end of file
+pulumi_github>=5.20.0
+black>=24.3
```

Comment:
How about using [ruff](https://docs.astral.sh/ruff/) instead?

Reply:
Sure! It really looks great!
I updated the `requirements.txt` and ran `ruff` on the dir, but it was already properly formatted, so it produced no changes.